src/middlewares/auth.js

```diff
@@ -1,8 +1,38 @@
+const User = require('../db/userModel')
+const bcrypt = require('bcryptjs')
+
 const auth = async (req, res, next) => {
-    if (!req.session.user) {
-        return res.status(401).redirect('/login')
+    try {
+        if (req.header('Authorization')) {
```

Comment:
Basic Authentication is less secure. Consider implementing token-based authentication (e.g., JWT) for enhanced security.

Reply:
Sure, this will be addressed in the future update, current implementation is intended to provide basic authentication.